index.js

```diff
@@ -1,25 +1,27 @@
 var _ = require('lodash');
 var loaderUtils = require('loader-utils');
 
-function getOptions(context) {
-  if (context.options && context.options.ejsLoader) {
-    return context.options.ejsLoader;
-  }
-  return {};
-}
+class UnsupportedConfiguration extends Error {}
 
 module.exports = function(source) {
   this.cacheable && this.cacheable();
-  var query = loaderUtils.parseQuery(this.query);
-  var options = getOptions(this);
+  var options = loaderUtils.getOptions(this);
+
+  if(!options.variable){
+    throw new UnsupportedConfiguration(`
+      To support ES Modules, the 'variable' option must be passed to avoid 'with' statements 
+      in the compiled template to be strict mode compatible. 
+      Please see https://github.com/lodash/lodash/issues/3709#issuecomment-375898111
+    `)
```

Comment:
Recommended: extract magic string to property of `UnsupportedConfiguration` or a constant and use that in the test, too.

Reply:
same lines as https://github.com/groveco/ejs-loader/pull/2#discussion_r433501567. I'm good with doing it but not sure if the pain is worth the benefit of checking specific exception casting.